ScoutSuite/output/data/inc-scoutsuite/pagination.js

```diff
@@ -0,0 +1,162 @@
+var defaultPageSize = 2
+var regionPageSize = 1
```

Comment:
Can we use  `const`?
```suggestion
const regionPageSize = 1
```